src/main/java/io/dropwizard/sharding/DBShardingBundle.java

```diff
@@ -79,6 +79,14 @@ public DBShardingBundle(String dbNamespace, String classPathPrefix) {
         init(inEntities);
     }
 
+    public DBShardingBundle(String dbNamespace, List<String> classPathPrefixList) {
```

Comment:
Probably better to add a var-arg constructor as well and route the the earlier constructor through that. Route everything through this finally.